Plugins/Statistics/GitStatistics/PieChart/PieChart3D.cs

```diff
@@ -851,7 +851,7 @@ protected virtual void InitializePieSlices()
         /// </returns>
         protected SizeF GetSliceDisplacement(float angle, float displacementFactor)
         {
-            Debug.Assert(displacementFactor > 0F && displacementFactor <= 1F, "displacementFactor > 0F && displacementFactor <= 1F");
+            Debug.Assert(displacementFactor is > 0F and <= 1F, "displacementFactor > 0F && displacementFactor <= 1F");
```

Comment:
💡 Consider using `IsDisplacementValid` instead of duplicating the check.

```suggestion
            Debug.Assert(IsDisplacementValid(displacementFactor), "IsDisplacementValid(displacementFactor)");
```

Reply:
IsDisplacementValid() is not the same but ==0F is handled anyway so I will change